### PR TITLE
Clustered: simplify public pull secrets

### PR DESCRIPTION
**Pull request by jenniferplusplus**
Crane is not actually required when using the InfluxData public image registry. It seems like referring to it before the public/private branch in the docs can cause some confusion. This moves all of the crane steps into the private branch, where it's needed.

- [x] Signed the [InfluxData CLA](https://www.influxdata.com/legal/cla/)
  ([if necessary](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#sign-the-influxdata-cla))
- [ ] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/clustered/install/configure-cluster/directly.md [hunk 1]
```diff
@@ -145,8 +145,58 @@ When pulling InfluxDB Clustered images, there are two main scenarios:
 - You run in an environment with no network interfaces ("air-gapped") and you
   can only access a private container registry.
 
-In both scenarios, you need a valid container registry secret file.
-Use [crane](https://github.com/google/go-containerregistry/tree/main/cmd/crane) to create a container registry secret file.
+In both scenarios, you need a valid pull secret.
```
Comment (jstirnaman) [inline content/influxdb/clustered/install/configure-cluster/directly.md:148]:
```suggestion
In both scenarios, you need a valid _pull secret_.
```

File: content/influxdb/clustered/install/configure-cluster/directly.md [hunk 2]
```diff
@@ -145,8 +145,58 @@ When pulling InfluxDB Clustered images, there are two main scenarios:
...
+1.  Copy the images from the InfluxDB registry to your own private registry.
+2.  Configure your `AppInstance` resource with a reference to your private
+    registry name.
+3.  Provide credentials to your private registry.
+
+##### Copy the images
+
+We recommend using [crane](https://github.com/google/go-containerregistry/tree/main/cmd/crane) to copy images into your private registry.
 
 1.  [Install crane](https://github.com/google/go-containerregistry/tree/main/cmd/crane#installation)
```
Comment (jstirnaman) [inline content/influxdb/clustered/install/configure-cluster/directly.md:201]:
```suggestion
1.  [Install crane](https://github.com/google/go-containerregistry/tree/main/cmd/crane#installation) for your system.
```

File: content/influxdb/clustered/install/configure-cluster/directly.md [hunk 3]
```diff
@@ -212,53 +262,6 @@ If there’s a problem with the Docker configuration, crane won't retrieve the m
...
-If your Kubernetes cluster can't use a public network to download container images
-from our container registry, do the following:
-
-1.  Copy the images from the InfluxDB registry to your own private registry.
-2.  Configure your `AppInstance` resource with a reference to your private
-    registry name.
-3.  Provide credentials to your private registry.
-
 The list of images that you need to copy is included in the package metadata.
 You can obtain it with any standard OCI image inspection tool. For example:
```
Comment (jstirnaman) [inline content/influxdb/clustered/install/configure-cluster/directly.md:266]:
```suggestion
You can obtain it with any standard OCI image inspection tool--for example:
```

**Review by jstirnaman — CHANGES_REQUESTED** [review]
A few small changes

**Comment by jenniferplusplus** [comment]
@jstirnaman is there anything else I'm missing?

**Review by jstirnaman — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**